src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -6491,53 +6464,44 @@ namespace VariableSpeedCoils {
 
 			}
 
-			if ( CBF < 0.001 ) {
-				SHRCalc = 1.0;
-				LoopOn = false;
-			}
-			else {
```

Comment:
The problem is here where if CBF is small, the SHR is set to 1. This is just wrong and SHR has nothing to do with coil bypass factor. This created a discontinuity that caused RootFinder to fail on max iterations.